modules/javafx.web/src/main/java/com/sun/webkit/Utilities.java

```diff
@@ -119,17 +116,6 @@ private static Object fwkInvokeWithContext(final Method method,
             });
         }
 
-        try {
-            return AccessController.doPrivileged((PrivilegedExceptionAction<Object>)
-                    () -> MethodHelper.invoke(method, instance, args), acc);
-        } catch (PrivilegedActionException ex) {
-            Throwable cause = ex.getCause();
-            if (cause == null)
-                cause = ex;
-            else if (cause instanceof InvocationTargetException
-                && cause.getCause() != null)
-                cause = cause.getCause();
-            throw cause;
-        }
+        return MethodHelper.invoke(method, instance, args);
```

Comment:
The removal of the try/catch is causing a test failure in `JavaScriptBridgeTest` because `InvocationTargetException` is now thrown rather than `InvocationTargetException::getCause`.

```suggestion
        try {
            return MethodHelper.invoke(method, instance, args);
        } catch (InvocationTargetException ex) {
            Throwable cause = ex.getCause();
            throw cause != null ? cause : ex;
        }
```

Reply:
changed as suggested.
is this test being run as part of GHA or the headful test suite?